### PR TITLE
VTN-32314: upgrade materialUI, react, redux, storybook, babel

### PR DESCRIPTION
**Pull request by seta-tuha**

### CODE REVIEW

File: packages/veritone-react-common/src/styles/modules/muiTypography.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// https://github.com/muicss/mui/blob/master/src/sass/mui/mixins/_typography.scss
+
+export const muiTextStyles = {
```
Comment (MmtBkn) [inline packages/veritone-react-common/src/styles/modules/muiTypography.js]: Set this props to theme

File: packages/veritone-react-common/.babelrc [hunk 2]
```diff
@@ -1,7 +1,42 @@
 {
+  "presets": [
+    "@babel/preset-env",
```
Comment (MmtBkn) [inline packages/veritone-react-common/.babelrc]: Add @babel/preset-typescript

**Comment by MmtBkn** [comment]
@seta-tuha this task definition doesn't include upgrading redux / babel. Please be careful with the scope of the task next time.

**Comment by seta-tuha** [comment]
> @seta-tuha this task definition doesn't include upgrading redux / babel. Please be careful with the scope of the task next time.

I got it. I will be paying more attention

File: packages/veritone-react-common/src/styles/modules/muiTypography.js (hunk 1 above)
Comment (seta-tuha) [reply packages/veritone-react-common/src/styles/modules/muiTypography.js]: work like this?
[https://material-ui.com/customization/theming/#custom-variables](url)

**Pull request merged**